src/Common/src/Interop/Windows/Interop.BOOLEAN.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal struct BOOLEAN
```

Comment:
I would think that 
```
    internal enum BOOL : int
    {
        FALSE = 0,
        TRUE = 1,
    }
```
should be sufficient given the limited use. Similar how Interop.BOOL.cs is done.

Reply:
The definition I have deals with non 0/1 values can be used just like a bool. What do we gain by making it an enum other than a simpler definition?

Reply:
You will get a simpler definition that was my point.

Reply:
Do you plan to address this? We are inconsistent today - both Interop.BOOL.cs and Interop.BOOLEAN.cs should use the same style.

Reply:
> Do you plan to address this? We are inconsistent today - both Interop.BOOL.cs and Interop.BOOLEAN.cs should use the same style.

As both you and Stephen like the enum version I'll switch it to that format.